### PR TITLE
E01H03-Buttons

### PR DESCRIPTION
**Pull request by Claurian**
se agrega componente de botones

### CODE REVIEW

**Comment by amolinerostgroup** [comment]
Can you please add the E01H04-Modals string to the PR title please?

File: src/components/Buttons/fonts/Montserrat.zip
Comment (amolinerostgroup) [inline src/components/Buttons/fonts/Montserrat.zip:1]: Is this the Roboto font?

File: src/components/Buttons/Button.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import React from "react";
```
Comment (amolinerostgroup) [inline src/components/Buttons/Button.js:1]: Can you please add a header with the description the author an the date of creation of this component please?

**Review by amolinerostgroup — CHANGES_REQUESTED** [review]

File: src/components/Buttons/fonts/Montserrat.zip
Comment (Claurian) [reply src/components/Buttons/fonts/Montserrat.zip:1]: The font for the text on the buttons is Monserrat, right?

File: src/components/Buttons/Button.js (hunk 1 above)
Comment (Claurian) [reply src/components/Buttons/Button.js:1]: ready.

**Review by amolinerostgroup — APPROVED** [review]

**Review by AZavalaG — COMMENTED** [review]
Import Fonts in css not in static files and manage PR to development, not production

**Comment by AZavalaG** [comment]
Still having fonts folder, please don not have static font files, all fonts must be imported from google fonts

**Pull request merged**